lib/extend.js

```diff
@@ -75,7 +76,7 @@ var extend = function extend() {
 /**
  * @public
  */
-extend.version = '1.1.3';
```

Comment:
Cancel version variable export is another good choice! Because I don't know why author export that, so I won't delete it directly. This pull aim to get/set version just only one place and remind author this place version is too old, our purpose are same.